src/common/state/redis.cc

```diff
@@ -1368,6 +1376,22 @@ void redis_local_scheduler_table_send_info(TableCallbackData *callback_data) {
   }
 }
 
+void redis_local_scheduler_table_disconnect(DBHandle *db) {
+  flatbuffers::FlatBufferBuilder fbb;
+  LocalSchedulerInfoMessageBuilder builder(fbb);
```

Comment:
Everywhere else I think we are constructing these messages with just a single big call to the `CreateLocalSchedulerInfoMessage ` method (or equivalent). Any particular reason for doing it this way instead? Just curious.

Reply:
Hmm I did this to make it explicit that the other fields are unset, but we could also set them to an empty `LocalSchedulerInfo`. Do you have a preference?

Reply:
Oh I see, I don't have a preference, the current way seems fine to me.

Reply:
Ok, it turns out this was a bug. It was causing the local scheduler valgrind error that we've been seeing. Hopefully fixed by #1037.

It was causing an assertion failure 

```
local_scheduler_tests: /home/travis/build/ray-project/ray/python/ray/core/flatbuffers_ep-prefix/src/flatbuffers_ep-install/include/flatbuffers/flatbuffers.h:890: void flatbuffers::FlatBufferBuilder::NotNested(): Assertion `!nested' failed.
```

in the test

```
bash ../../../src/local_scheduler/test/run_valgrind.sh
```